### ISSUE TITLE
Can't change symbol table to "\"

### ISSUE THREAD

**Issue by csadams** [asdil12/pymultimonaprs#16]
I'm having issues changing my symbol table to "\" (backslash) instead of the default "/" (slash).  When I change it in pymultimonaprs.json, and run "sudo pymultimonaprs", it gives me an error: "ValueError Invalid control character at: line 20 column 16 (char 342)".  I would like to change my symbol to the "Gate" symbol instead of the default "Gate-HF" symbol, since I'm operating a VHF iGate, not HF. I'm running it on a Raspberry Pi 2 Model B, with Raspbian Wheezy. Let me know if you need more info.  Thanks for the great software! -Chris KJ6WEG

**Comment by asdil12** [comment]
As a backslash is the excape character, you need to write `\\`

**Comment by csadams** [comment]
Thanks for the quick response!  
I tried it with "table": "\\", and it gave me the black diamond on APRS.fi, but it doesn't have the "I" symbol in it.  After some digging through the packets from other igates, I see that they use table="I" and symbol="&" to get the common iGate symbol (black diamond with "I" overlay).  I confirmed this as valid in the APRS protocol reference: http://www.aprs.org/doc/APRS101.PDF... and learned that the table code can be replaced with the number or letter to overlay on the primary table symbol. It's all very confusing for a noob like me, but seems to work.  Thanks again.
